### PR TITLE
Add working integration with trello

### PR DESCRIPTION
**Pull request by mksochota16**

### CODE REVIEW

**Comment by RouNNdeL** [comment]
Please rebase on top of master

File: src/trello_conn.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        board = client.get_board(self.board_id)
+        lists = board.all_lists()
+        cards = []
+        for trello_list in lists:
+            cards.extend(trello_list.list_cards())
+        return cards
+
+
+if __name__ == "__main__":
+    print("FINISH")
```
Comment (RouNNdeL) [inline src/trello_conn.py:56]: what is that?

File: src/trello_conn.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        selected_list = None
+        for trello_list in lists:
+            # if None return first, unless it is archived
+            if (list_name is None and not trello_list.closed) or trello_list.name == list_name:
+                selected_list = trello_list
+                break
+        if selected_list is not None:
+            selected_list.add_card(card_name, card_description)
+        else:
+            print(f"No list found with a given name: {list_name}")
```
Comment (RouNNdeL) [inline src/trello_conn.py:42]: Use `logging` from Python

File: src/trello_conn.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from trello import TrelloClient
+import requests
+
+
+def card_already_in_trello(card_id):
+    with open("trello_cards") as file:
```
Comment (RouNNdeL) [inline src/trello_conn.py:6]: Move filename to config

File: src/trello_conn.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        lines = file.readlines()
+    if f"{card_id}\n" in lines:
+        return True
+    else:
+        with open('trello_cards', 'a') as file:
+            file.write(f'{card_id}\n')
+        return False
```
Comment (RouNNdeL) [inline src/trello_conn.py:13]: This reads the whole file every time a single card is check, please read it only once. Also make this part of a class.

**Review by RouNNdeL — CHANGES_REQUESTED** [review]

File: src/trello_conn.py (hunk 2 above)
Comment (mksochota16) [reply src/trello_conn.py:42]: From what I can see, that would be the only use of ```logging``` from python in this project. Do you really want me to add a log file or for example throw an exception here?

File: trello_cards [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+6106998
+6106924
+6105850
+6102060
+6101015
+6097488
+6096040
+6096007
+6095146
+6094220
```
[the remark this answers is not in the document]
Comment (RouNNdeL) [reply trello_cards:25]: file

File: src/trello_conn.py (hunk 2 above)
Comment (RouNNdeL) [reply src/trello_conn.py:42]: Not the [first](https://github.com/RouNNdeL/aiesec-reception-tool/blob/e3e079881e6e577c3bc7e052f33aeb3b5d88e22e/src/main.py#L51), certainly not the last

Comment (RouNNdeL) [reply src/trello_conn.py:42]: But the exception fits better here, you are right

**Pull request closed without merging**